setupbase.py

```diff
@@ -210,7 +210,7 @@ def run(self):
             hashes[style] = hashlib.md5(data).hexdigest()
 
         # Now it's time to patch "iconic_font.py":
```

Comment:
Should be `__init__.py`

Reply:
Fixed at 0ae5d6bcf298f30747ad6086f19f25dfa08161c9